### PR TITLE
add command to set WEBOTS_SAFE_MODE in windows

### PR DESCRIPTION
**Pull request by Behrouz-m**
**Documentation**
If this pull-request changes the doc, add the link to the related page, including the `?version=BRANCH_NAME`, such as:
https://cyberbotics.com/doc/guide/starting-webots?version=R2021a

### CODE REVIEW

File: docs/guide/starting-webots.md
Comment (omichel) [inline docs/guide/starting-webots.md:131]:
```suggestion
From the Windows graphical user interface:
```

File: docs/guide/starting-webots.md
Comment (omichel) [inline docs/guide/starting-webots.md:136]:
```suggestion
From the `cmd` command prompt:
```

**Review by omichel — CHANGES_REQUESTED** [review]
Thank you for this contribution. I only have a couple of styling requests.

**Comment by Behrouz-m** [comment]
@omichel please check it. thanks for the suggestion.

**Review by omichel — APPROVED** [review]
Thank you!

**Comment by omichel** [comment]
@Behrouz-m: I just sent you an invite to become a member of the Committers group. Once accepted, you should be able to merge this PR.

**Comment by Behrouz-m** [comment]
> 
> 
> @Behrouz-m: I just sent you an invite to become a member of the Committers group. Once accepted, you should be able to merge this PR.

Thank you. It was unexpected :)
I am happy to be a member of your team.

**Comment by omichel** [comment]
> Thank you. It was unexpected :)
I am happy to be a member of your team.

You are welcome!

**Pull request merged**
